dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -243,7 +256,10 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
     this.setState({ releaseName: e.currentTarget.value });
   };
   public handleChartVersionChange = (e: React.FormEvent<HTMLSelectElement>) => {
-    this.props.push(`/apps/new/${this.props.chartID}/versions/${e.currentTarget.value}`);
+    const { chartID, getChartVersion, getChartValues } = this.props;
+
+    getChartVersion(chartID, e.currentTarget.value);
```

Comment:
we should only do this in AppEdit, if we do it in AppNew the version in the URL won't reflect what's in the state.